[2024-07-30T23:53:27.553+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T23:53:27.576+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Pipeline.test_postgres_connection scheduled__2024-07-30T23:52:00+00:00 [queued]>
[2024-07-30T23:53:27.584+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Pipeline.test_postgres_connection scheduled__2024-07-30T23:52:00+00:00 [queued]>
[2024-07-30T23:53:27.585+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-30T23:53:27.596+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_postgres_connection> on 2024-07-30 23:52:00+00:00
[2024-07-30T23:53:27.604+0000] {standard_task_runner.py:64} INFO - Started process 3811 to run task
[2024-07-30T23:53:27.608+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Pipeline', 'test_postgres_connection', 'scheduled__2024-07-30T23:52:00+00:00', '--job-id', '540', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpk58n3yid']
[2024-07-30T23:53:27.611+0000] {standard_task_runner.py:91} INFO - Job 540: Subtask test_postgres_connection
[2024-07-30T23:53:27.659+0000] {task_command.py:426} INFO - Running <TaskInstance: Pipeline.test_postgres_connection scheduled__2024-07-30T23:52:00+00:00 [running]> on host f17ca52b44b4
[2024-07-30T23:53:27.738+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Pipeline' AIRFLOW_CTX_TASK_ID='test_postgres_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T23:52:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-30T23:52:00+00:00'
[2024-07-30T23:53:27.740+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T23:53:27.760+0000] {base.py:84} INFO - Using connection ID 'connect_db' for task execution.
[2024-07-30T23:53:27.805+0000] {logging_mixin.py:188} INFO - Connection test result: (1,)
[2024-07-30T23:53:27.805+0000] {logging_mixin.py:188} INFO - Conexi√≥n exitosa a PostgreSQL.
[2024-07-30T23:53:27.807+0000] {logging_mixin.py:188} INFO - Nombre de la base de datos: accidente_de_trafico_2021_2022
[2024-07-30T23:53:27.808+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-30T23:53:27.809+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T23:53:27.818+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Pipeline, task_id=test_postgres_connection, run_id=scheduled__2024-07-30T23:52:00+00:00, execution_date=20240730T235200, start_date=20240730T235327, end_date=20240730T235327
[2024-07-30T23:53:27.862+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-30T23:53:27.880+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T23:53:27.882+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
